bears/general/URLBear.py

```diff
@@ -18,6 +18,26 @@ class LINK_CONTEXT(Flag):
     pip_vcs_url = 2
 
 
+class URLResult(HiddenResult):
+
+    def __init__(self, line_number, link, code, context):
+        if not isinstance(line_number, int):
+            raise ValueError('line_number should be an integer')
+        if not isinstance(link, str):
+            raise ValueError('link should be a string')
+        if not (isinstance(code, int) or code is None):
+            raise ValueError('code should be an integer or None')
+        if not isinstance(context, LINK_CONTEXT):
+            raise ValueError('context should be a URLBear.LINK_CONTEXT')
+
+        HiddenResult.__init__(self, URLBear, [line_number, link, code,
```

Comment:
IMO dont even bother calling `HiddenResult.__init__` .  Just call `Result.__init__(...)`.

Or, do https://github.com/coala/coala/issues/4454 first ;-)

`message` can be "{url} is HTTP {status}"